source/index.js

```diff
@@ -36,15 +37,43 @@ var likely = function (node, options) {
 };
 
 /**
+ * Deprecated. Will be removed in 3.0
```

Comment:
BTW, this can be expressed with [`@deprecated` ](http://usejsdoc.org/tags-deprecated.html) (sorry for noticing late).